### ISSUE TITLE
android-maps-utils

### ISSUE THREAD

**Issue by osedok** [ngageoint/geopackage-android-map#14]
@bosborn Hi Brian,
Is there any chance that you can upgrade android-maps-utils to the latest sources available. I am using some parts of that app to import KML into my app - converting it first into geojson from KmlLayer object - do you recommend any other way of doing it?

If you compare e.g. KmlPoint class from version 0.5 (embedded in geopackage-map) to https://github.com/googlemaps/android-maps-utils/blob/master/library/src/com/google/maps/android/data/kml/KmlPoint.java you will notice quite a few differences - for me the most important thing is constructor:

```
public KmlPoint(LatLng coordinates, Double altitude) {
        super(coordinates);

        this.mAltitude = altitude;
    }

```
I would appreciate your opinion.

Kind Regards,
Andrzej

**Comment by bosborn** [comment]
I will definitely update to the most recent released version, which [is still currently 0.5](https://github.com/googlemaps/android-maps-utils/releases).  The changes [you are referencing](https://github.com/googlemaps/android-maps-utils/commits/master/library/src/com/google/maps/android/data/kml/KmlPoint.java) were made a month after the 0.5 release and have not yet been released.  We can't rely on an unreleased library version.

I am not sure why they've waited so long between releases.  I'm not sure if there are changes you can temporarily pull into your own code while avoiding conflicts.  You can also fork your own version of this library and make those dependency changes.

**Comment by osedok** [comment]
Hi Brian, I do understand your concerns, yes the 0.5 is the only release so far - I am already using my own fork of it in all my projects.  Thanks for: "I will definitely update to the most recent released version..." - I will be watching the space.

**Comment by bosborn** [comment]
Noticed they [released a few new versions](https://github.com/googlemaps/android-maps-utils/releases).
I [updated the android-map-utils version](https://github.com/ngageoint/geopackage-android-map/blob/5b4743f6e7ab8e44c7d2d974e8161e6cc0a1c132/geopackage-map/build.gradle#L165) to 0.6.2 for the next release.

**Comment by bosborn** [comment]
Released [3.4.0](https://github.com/ngageoint/geopackage-android-map/releases/tag/3.4.0) with an [android-map-utils 0.6.2](https://github.com/googlemaps/android-maps-utils/releases/tag/0.6.2) dependency.
